nubis/files/nubis-lib.sh

```diff
@@ -5,12 +5,14 @@
 #
 # [ -e /usr/local/lib/nubis/nubis-lib.sh ] && . /usr/local/lib/nubis/nubis-lib.sh || exit 1
 
-NUBIS_ARENA=$(nubis-metadata NUBIS_ARENA)
-NUBIS_STACK=$(nubis-metadata NUBIS_STACK)
+eval "$(nubis-metadata)"
```

Comment:
I generally recommend being explicit in what metadata we need instead of the eval trick. As in this case:

```
NUBIS_ARENA=$(nubis-metadata NUBIS_ARENA)
NUBIS_STACK=$(nubis-metadata NUBIS_STACK)
NUBIS_ENVIRONMENT=$(nubis-metadata NUBIS_ENVIRONMENT)
NUBIS_PROJECT=$(nubis-metadata NUBIS_PROJECT)
NUBIS_PURPOSE=$(nubis-metadata NUBIS_PURPOSE)
```